ingestion/src/metadata/ingestion/source/database/sql_column_handler.py

```diff
@@ -96,7 +96,11 @@ def _process_col_type(self, column: dict, schema: str) -> Tuple:
             col_type = ColumnTypeParser.get_column_type(column["type"])
             # For arrays, we'll get the item type if possible, or parse the string representation of the column
             # if SQLAlchemy does not provide any further information
-            if col_type == "ARRAY" and getattr(column["type"], "item_type"):
+            if (
```

Comment:
```suggestion
if col_type == "ARRAY" and getattr(column["type"], "item_type", None)
```

we could use getattr with a default if we don't want 2 statements